scipy/signal/filter_design.py

```diff
@@ -311,6 +312,29 @@ def zpk2tf(z, p, k):
     else:
         b = k * poly(z)
     a = atleast_1d(poly(p))
+
+    # Use real output if possible.  Copied from numpy.poly, since
```

Comment:
This should be in [_numpy_compat.py](https://github.com/scipy/scipy/blob/master/scipy/_lib/_numpy_compat.py) then?